all-builds.nix

```diff
@@ -85,6 +95,14 @@ let
           ghcjs
           serverSkeletonExe
           ;
+      } // lib.optionalAttrs (system == "x86_64-linux") {
+        server =
+          if profiling
+          then {} # build with profiling has deps marked as broken (e.g. ‘th-orphans-0.13.7’)
```

Comment:
I was surprised to find this. 
I'd expect haskell packages to be contained only in the `exe` profiling build ?